backend/api/routers/pool_router.py

```
@@ -213,4 +217,86 @@ async def import_pool(
                 detail=f"Failed to import pool: {result.error}"
             )
     except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.get("/performance/iostat", response_model=Dict[str, Any])
+async def get_iostat(
+    pools: Optional[str] = Query(None, description="Comma-separated list of pools"),
+    interval: int = Query(1, description="Interval in seconds"),
+    count: int = Query(5, description="Number of samples"),
+    pool_service: PoolService = Depends(get_pool_service)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)